lib/data/datasources/question_service.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:job_interview_practice/data/models/question.dart';
+import 'package:job_interview_practice/data/repositories/question_repository_impl.dart';
+
+class QuestionService {
+
+  QuestionService(this.numberOfQuestions);
+
+  int           numberOfQuestions; // todo: numberofquestionをsettings pageから引っ張ってくる。
+  int           answeredQuestion;
+  QuestionModel nowQuestion;
```

Comment:
Usually the constructor would come before the declarations.  If you want to write code professional code, you might want to do that.